addon/components/single-document-details.hbs

```diff
@@ -198,13 +198,17 @@
               <span class="uk-width-expand">
                 {{~await (resolve-user file.createdByUser)~}}
               </span>
-              <button
+              <a
                 uk-icon="download"
                 class="uk-icon-link"
                 type="button"
```

Comment:
This can be removed i think since its not a button anymore.

Reply:
Done